test/winter_onboarding_2021/fleet_management_service/db/fleet_test.clj

```diff
@@ -17,7 +40,39 @@
           user-id (:users/id (user-db/create user))
           fleet {:name "Azkaban Fleet 1"
                  :created-by user-id}]
-      (fleet-db/create fleet)
       (is (= #:fleets{:name "Azkaban Fleet 1"
-                      :created-by user-id} (select-keys (first (db-core/query! ["select * from fleets;"]))
-                                                        [:fleets/created-by :fleets/name]))))))
+                      :created-by user-id} (select-keys (fleet-db/create fleet)
+                                                        [:fleets/name :fleets/created-by]))))))
+
+(deftest select-fleets-for-admin
+  (testing "Should fetch us a list of first 10 fleets related to an admin user"
+    (let [{:keys [user fleets]} (seed-user-fleets-db 5)
+          _ (seed-user-fleets-db 5)
+          db-fleets (mapv dissoc-irrelevant-keys (fleet-db/select-fleets-for-admin (:users/id user) 0 10))]
+      (is (= 5 (count fleets)))
+      (is (= fleets db-fleets)))))
+
+(deftest pagination
```

Comment:
I think, a better way to test pagination would be to add not a multiple of 10 number of fleets lets say 13.
Then assert on the following
- First page contains 10 fleets
- Second page also contain 3 fleets